frontend/public/components/image-stream.tsx

```diff
@@ -325,85 +348,115 @@ export const ImageStreamsDetailsPage: React.FCC = (props) => (
 );
 ImageStreamsDetailsPage.displayName = 'ImageStreamsDetailsPage';
 
-const tableColumnClasses = [
-  '',
-  '',
-  'pf-m-hidden pf-m-visible-on-md',
-  'pf-m-hidden pf-m-visible-on-lg',
-  Kebab.columnClass,
-];
+const getDataViewRows: GetDataViewRows<K8sResourceKind, undefined> = (data, columns) => {
+  return data.map(({ obj: imageStream }) => {
+    const { name, namespace, labels, creationTimestamp } = imageStream.metadata;
+    const context = { [ImageStreamsReference]: imageStream };
 
-const ImageStreamsTableRow: React.FC<RowFunctionArgs<K8sResourceKind>> = ({ obj }) => {
-  return (
-    <>
-      <TableData className={tableColumnClasses[0]}>
-        <ResourceLink
-          kind={ImageStreamsReference}
-          name={obj.metadata.name}
-          namespace={obj.metadata.namespace}
-        />
-      </TableData>
-      <TableData className={css(tableColumnClasses[1], 'co-break-word')} columnID="namespace">
-        <ResourceLink kind="Namespace" name={obj.metadata.namespace} />
-      </TableData>
-      <TableData className={tableColumnClasses[2]}>
-        <LabelList kind={ImageStreamsReference} labels={obj.metadata.labels} />
-      </TableData>
-      <TableData className={tableColumnClasses[3]}>
-        <Timestamp timestamp={obj.metadata.creationTimestamp} />
-      </TableData>
-      <TableData className={tableColumnClasses[4]}>
-        <ResourceKebab actions={menuActions} kind={ImageStreamsReference} resource={obj} />
-      </TableData>
-    </>
-  );
+    const rowCells = {
+      [tableColumnInfo[0].id]: {
+        cell: <ResourceLink kind={ImageStreamsReference} name={name} namespace={namespace} />,
+        props: { ...getNameCellProps(name), width: 10, modifier: 'nowrap' },
+      },
+      [tableColumnInfo[1].id]: {
+        cell: <ResourceLink kind="Namespace" name={namespace} />,
+      },
+      [tableColumnInfo[2].id]: {
+        cell: <LabelList kind={ImageStreamsReference} labels={labels} />,
+        props: {
+          width: 40,
+          modifier: 'nowrap',
+        },
+      },
+      [tableColumnInfo[3].id]: {
+        cell: <Timestamp timestamp={creationTimestamp} />,
+        props: {
+          width: 10,
+          modifier: 'nowrap',
+        },
+      },
+      [tableColumnInfo[4].id]: {
+        cell: <LazyActionMenu context={context} />,
```

Comment:
kebab menu is not showing up on ImageStream list
<img width="1247" height="371" alt="ImageStreamsList" src="https://github.com/user-attachments/assets/0fc56618-b404-4c45-8761-019079ac6613" />